src/TestSuite/IntegrationTestCase.php

```diff
@@ -343,11 +367,11 @@ protected function _buildRequest($url, $method, $data)
         ];
         $session = Session::create($sessionConfig);
         $session->write($this->_session);
-
         list ($url, $query) = $this->_url($url);
+
         $props = [
             'url' => $url,
-            'post' => $data,
+            'post' => $this->_addTokens($url, $method, $data),
```

Comment:
Yes because it's more consistent in my opinion. Let the user decide what kind of data he wants to inject to the request instead of force the `_csrfToken`. The forced `_csrfToken` may ends up with some failing test cases in existing applications and would be a BC break if you check against the number of setted cookies or similar things for whatever reason.